core/src/main/java/org/elasticsearch/search/aggregations/bucket/DeferringBucketCollector.java

```diff
@@ -54,6 +54,16 @@ public Aggregator wrap(final Aggregator in) {
         return new WrappedAggregator(in);
     }
 
+    /**
+     * Unwrap an aggregator. Used for testing.
+     */
+    public static Aggregator unwrap(Aggregator in) {
```

Comment:
I'm not super comfortable about exposing this but I can't think of a much better way to actually close the aggregations during testing without reworking aggregations to make parent aggs close their subaggregations.